CriticalMass/de.lproj/Localizable.strings

```diff
@@ -105,14 +105,27 @@
 /* No comment provided by engineer. */
 "settings.socialMedia" = "Social Media";
 
-"settings.openSource" = "This is an open source project. Feel free to support us with coding for iOS or Web:";
+/* No comment provided by engineer. */
+"settings.opensource.title" = "Vermisst du Funktionen oder willst einen Bug fixen?";
+
+/* No comment provided by engineer. */
+"settings.opensource.detail" = "Critical Maps ist ein Open Source Projekt und wir feuen uns über Entwickler*innen die an Critical Maps mitarbeiten wollen.";
+
+/* No comment provided by engineer. */
+"settings.opensource.action" = "PROJEKT ANSEHEN";
```

Comment:
Can we change to normal case "Projekt ansehen" and uppercase the string in the code? If design changes, we don't need to touch the strings imo

Reply:
Yes. Good Idea!